TechDoc-Assistance-Program.md

```diff
@@ -0,0 +1,55 @@
+
+## Phase 1: Documentation Analysis
+
+A technical writer (on CNCF staff or on contract) analyzes the documentation. Based on the standards developed by the CNCF Tech Docs program, the writer:
+
+1. Estimates the maturity level of the documentation compared to the current or desired maturity level of the software project, based on a number of criteria in three categories, using a rubric developed by CNCF. The categories are:
+    1. Project documentation: The end-user documentation for the project's work product, typically (but not always) an application, API, protocol, or some other software product.
+    1. Contributor documentation: Documentation about the project, aimed at project contributors and describing procedures, infrastructure, and customs for doing project work. This includes artifacts that define procedures and governance; recruit, orient, and train project contributors; and name responsible parties (project leaders, often generically called *maintainers*).
+    1. Website: The technical infrastructure behind the documentation and the project's web presence, including website generation, versioning, SEO, analytics, and security.
+1. Collaborates with project leadership to identify user roles and objectives for software users. 
+1. Proposes changes, if necessary, to the organization and content of the documentation to close gaps with the proposed maturity level.
+1. Writes an implementation plan describing improvements that address the gaps identified in the analysis.
+
+## Phase 2: Backlog Creation
+
+Once a high-level improvement plan has been written and approved, the tech writer analyzes the proposed changes to create a backlog of actionable, individual writing assignments and other tasks to improve the documentation. These tasks should have two primary characteristics:
+- As much as possible, they should be independent of each other so they can be completed in any order by any combination of contributors; and
+- They should be time-constrained. If possible, large tasks should be broken down into smaller, independent tasks that require hours, not days, to complete.
+
+## Phase 3: Documentation Improvement
+
+Community members work on the issues created in the previous phase. Ideally, tech writers are available to advise contributors and edit work, especially if the contributing community members are not trained technical writers. Remember that the training courses in Phase 0 are available to prepare contributors with general knowledge of the process.
+
+We know that recruiting contributors to write documentation can be difficult. While this is largely the responsibility of the project leadership, CNCF is actively working on ways to encourage doc contributions. For example, creating a backlog of time-bounded issues is an attempt to lower barriers, both psychological and logistical, to documentation creation and maintenance. 
```

Comment:
@nate-double-u - Do we want to mention mentorship opportunities twice(?) a year?